src/makeWrapperNative.sh

```diff
@@ -0,0 +1,88 @@
+#! @bash@
+
+# SPDX-License-Identifier: AGPL-3.0-or-later
+# Copyright (c) 2025 midischwarz12
+
+set -euo pipefail
+
+PYTHON=@python@
+RUNNER=@runner@
+
+WRAPPER_RUNNER=$RUNNER exec "$PYTHON" - <<'PY'
+import argparse
+import os
+import pathlib
+import shutil
+import struct
+import sys
+
+MARKER = b"WRAPCFG1"
+
+
+def main():
```

Comment:
What's the purpose of this? This is a shell script with python completely baked into it.